addons/point_of_sale/static/src/xml/Screens/ProductScreen/ProductScreen.xml

```diff
@@ -20,6 +20,7 @@
                         <div class="subpads">
                             <ActionpadWidget
                                 partner="partner"
+                                actionType="'payment'"
```

Comment:
We have to wait the pr before we push this one.
https://github.com/odoo/odoo/pull/120846